### PR TITLE
Adding support for rendering frame level attributes in ETA

### PR DESCRIPTION
**Pull request by brimoor**
Example:

Screen Shot 2019-04-02 at 12 25 00 AM

### CODE REVIEW

File: eta/core/annotations.py [hunk 1]
```diff
@@ -1,11 +1,7 @@
 '''
 Core utilities for rendering annotations on media.
 
-@todo support for rendering video-level attributes
```
Comment (brimoor) [inline eta/core/annotations.py]: @yashbhalgat note that I went ahead and supported the other two todo items

File: eta/core/models.py [hunk 2]
```diff
@@ -927,13 +926,7 @@ class PickledModel(PublishedModel):
     '''Class that can load a published model stored as a .pkl file.'''
 
     def _load(self):
-        try:
```
Comment (brimoor) [inline eta/core/models.py]: @yashbhalgat note that I moved this `try-except` to a `eta.core.serial.read_pickle`. The idea is that we only want to implement reading pickle files once and then re-use it everywhere. You fixed this case, but there was a duplicate implementation of pickle reading in `Picklable.from_pickle`, so I fixed that.

File: eta/core/annotations.py (hunk 1 above)
Comment (yashbhalgat) [reply eta/core/annotations.py]: Yes.. This looks good now

File: eta/core/models.py (hunk 2 above)
Comment (yashbhalgat) [reply eta/core/models.py]: Good idea!

**Comment by jasoncorso** [comment]
Note that my preference for the video level attributes is for the box to
extend to the top and left boundaries of the frame.  Seems cleaner to me.

**Comment by brimoor** [comment]
I matched how it looks in player51. it is customizable in this code however- one can change paddings as desired

**Pull request merged**